odo/backends/tests/test_dask.py

```diff
@@ -3,14 +3,17 @@
 import pytest
 pytest.importorskip('dask')
```

Comment:
Should also skip on `dask.bag`.  It has some dependencies that aren't explicitly stated in dask proper.